lib/galaxy_test/api/test_tools.py

```diff
@@ -2573,6 +2573,24 @@ def test_metadata_validator_can_fail_on_deferred_input(self, history_id):
         job_details = self.dataset_populator.get_job_details(job_id=job_id).json()
         assert job_details["state"] == "failed"
 
+    @skip_without_tool("gx_allow_uri_if_protocol")
+    def test_allow_uri_if_protocol_on_deferred_input(self, history_id):
```

Comment:
I suppose it'll just work, but can you also test passing in a collection here?

Reply:
Done in https://github.com/galaxyproject/galaxy/pull/19077/commits/2bdcec0dda2d9e862fa7a043de6b0207223e43ca :)